database/migrations/2020_01_01_160000_create_search_volumes_table.php

```diff
@@ -10,6 +10,8 @@ class CreateSearchVolumesTable extends Migration
 {
     public function up()
     {
+        Schema::defaultStringLength(191);
```

Comment:
From memory, this should only be necessary for MariaDB or MySQL less than 5.7.7. I'm not sure how old that version is now or if you'll want to support it.

Furthermore, most [docs](https://laravel.com/docs/master/migrations#index-lengths-mysql-mariadb) recommend adding this in your `AppServiceProvider`, not packages. So it probably shouldn't go here.